tequilapi/endpoints/node.go

```diff
@@ -78,6 +80,32 @@ func (ne *NodeEndpoint) MonitoringAgentStatuses(c *gin.Context) {
 	utils.WriteAsJSON(contract.MonitoringAgentResponse{Statuses: res}, c.Writer)
 }
 
+// GetProviderSessions A list of sessions metrics during a period of time
+// swagger:operation GET /node/provider/sessions GetProviderSessions
+// ---
+// summary: Provides Node sessions data during a period of time
+// description: Node sessions metrics during a period of time
+// parameters:
+//   - in: query
+//     name: range
+//     description: period of time ("1d", "7d", "30d")
+//     type: string
+// responses:
+//   200:
+//     description: Provider sessions list
+//     schema:
+//       "$ref": "#/definitions/ProviderSessionsResponse"
+func (ne *NodeEndpoint) GetProviderSessions(c *gin.Context) {
+	rangeTime := c.Param("range")
```

Comment:
This seem to not be validated anywhere and is described as being `period of time ("1d", "7d", "30d")`. Even if it's checked on the service you're calling, it will then produce a `http.StatusInternalServerError` which is not right, it should be a Bad Request error.